.gitignore

```diff
@@ -1,2 +1,25 @@
 node_modules/
 *.log
```

Comment:
The initial `.gitignore` provided by Mintbean covers some of the files ignored below. This isn't an issue but something to note in case you want to change your dependencies down the line.